lib/util/cache/repository/impl/local.spec.ts

```diff
@@ -6,6 +9,21 @@ import { LocalRepoCache } from './local';
 
 jest.mock('../../../fs');
 
+const compress = promisify(zlib.brotliCompress);
+
+async function createCacheRecord(
+  data: RepoCacheData,
+  repository = 'some/repo'
+): Promise<RepoCacheRecord> {
+  const revision = CACHE_REVISION;
+  const jsonStr = JSON.stringify(data);
+  const hash = hasha(jsonStr, { algorithm: 'sha1' });
```

Comment:
shouldn't we use sha256 to be more safe?

Reply:
I believe it has nothing to do with encryption/security, I'm just using it the similar way it's being used in `git`

Reply:
git is also slowly transitioning to sha256 😉

Reply:
Haha, you got me